### PR TITLE
Small menubar fixes

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] Unify icon path code
- [x] disable double click so clicks just open / close the tab
- [x] fix icon disappearing on osx. we had this code to switch the colors but i think its more trouble than its worth.

### CODE REVIEW

File: shared/menubar/icons.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import * as NotificationTypes from '../constants/types/notifications'
+import {isDarwin, isWindows, isLinux} from '../constants/platform'
+
+const getIcons = (iconType: NotificationTypes.BadgeType, isBadged: boolean, isSystemDarkMode: boolean) => {
```
Comment (jzila) [inline shared/menubar/icons.tsx:4]: Why pass in `isSystemDarkMode` instead of importing it here?

Comment (chrisnojima) [reply shared/menubar/icons.tsx:4]: because this is used in the node and chrome contexts and i don't want to pull in all those deps

Comment (chrisnojima) [reply shared/menubar/icons.tsx:4]: and pulling it out of styles depends on timing which i don't want to race on

Comment (jzila) [reply shared/menubar/icons.tsx:4]: Got it, thanks.

**Pull request merged**